library/src/main/java/com/getstream/sdk/chat/StreamChat.java

```diff
@@ -170,10 +178,20 @@ public static FontsManager getFontsManager(){
     }
 
     public static synchronized boolean init(String apiKey, Context context) {
-        return init(apiKey, new ApiClientOptions(), context);
+        return init(apiKey, new ApiClientOptions(), context, new StreamNotificationsManager());
     }
 
-    public static synchronized boolean init(String apiKey, ApiClientOptions apiClientOptions, @NonNull Context context) {
+    public static synchronized boolean init(String apiKey,
+                                            ApiClientOptions apiClientOptions,
+                                            @NonNull Context context) {
+        return init(apiKey, apiClientOptions, context, new StreamNotificationsManager());
+    }
+
+    public static synchronized boolean init(String apiKey,
+                                            ApiClientOptions apiClientOptions,
+                                            @NonNull Context context,
+                                            NotificationsManager notificationsManager) {
```

Comment:
Let's use setter for `NotificationsManager`, it's a braking api change.
We're adding more modules right now (FontManager, NotificationsManager, etc). Later we will add `StreamChat.Builder` where all modules could be configured.

So it's better to break api once and later.

Reply:
@andriizhumela ^^^